testing/integration/tests/main.rs

```diff
@@ -814,6 +814,117 @@ fn test_oom2() {
     assert_eq!(res.msg_receipt.exit_code, ExitCode::SYS_ILLEGAL_INSTRUCTION);
 }
 
+#[test]
+fn test_oom3() {
+    // Instantiate tester
+    let mut tester = new_tester(
+        NetworkVersion::V18,
+        StateTreeVersion::V5,
+        MemoryBlockstore::default(),
+    )
+    .unwrap();
+
+    let sender: [Account; 1] = tester.create_accounts().unwrap();
+
+    let wasm_bin = HELLO_BINARY.unwrap();
```

Comment:
because it should do something other than oom to discriminate.
For the oom4 test i think we want a static alloc and a phony method invocation to discriminate.

Reply:
Makes sense. Let's document that and maybe give the test a better name to make that clear).